docs/src/components/Contributors/index.tsx

```diff
@@ -0,0 +1,64 @@
+import React from "react";
+import { useStaticQuery, graphql } from "gatsby";
+import Contributor from "./Contributor";
+
+const CURRENT_CONTRIBUTORS = [
+  "Victor Genaev",
+  "Luděk Vepřek",
+  "Aaron Collier",
+  "Matija Marohnić",
+  "William Kolmačka",
+  "Milan Seitler",
```

Comment:
nope, i think we should solve it on github somehow 🤔 . We cant fetch will because he does not have name, only login. Milan is not in contributors at all and that is probably because he did not contribute to project on github.